client/ayon_core/hosts/nuke/api/utils.py

```diff
@@ -142,3 +149,150 @@ def is_headless():
         bool: headless
     """
     return QtWidgets.QApplication.instance() is None
+
+
+def create_error_report(context):
```

Comment:
It's too bad this is in `nuke.api.utils` - this is really something that should be "easy" to retrieve from the publishing API and should not need a 'nuke specific implementation' I would say.

Anyway, docstring would be welcome.

Reply:
Done.

Reply:
Could this be at least renamed to `create_publish_error_report`?

Reply:
👆